.estafette.yaml

```diff
@@ -58,7 +61,7 @@ stages:
           timeoutSeconds: 180
     image: extensions/helm:dev
     action: test
-    prerelease: true
+    version: ${ESTAFETTE_BUILD_VERSION}
```

Comment:
I don't see this `version` parameter set in the other components manifest. Why do we need here?